Shapes.java

```diff
@@ -1,25 +1,23 @@
-import java.util.*;
+import java.until.*;
 
-public class Shapes{
-  public static void main(String[] args) {
-    Scanner input = new Scanner(System.in);
-    System.out.print("Enter the side length: ");
-    double side = input.nextDouble();
-    square(side);
-  }
+ public class Shapes{
+   public static void main(String[] args){
+     square();
+     triangle();
 
-  /*
-   * N: square
-   * P: to calculate the area of any square
-   * I: a side measure (double)
-   * R:
-   */
-  public static void square(double s){
-    //side times side
-    double a = s * s;
+     public static void square(){
+       double Squareside = 10;
+       double ans = Squareside*Squareside;
+       system.out.println(ans);
 
-    //generate some output
-    System.out.println("A square with side " + s + " has an area of " + a);
-  }//close square method
 
-}
+       public static void triangle(){
+         double H = 10;
```

Comment:
So the standard practice for variables names in Java is to make them `camelCase`, meaning lowercase, but subsequent words are capitalized. So good job with `ans2` on line 18, but that means you should change line 15-17, right?